test/integ_tests/device_creation.jl

```diff
@@ -5,9 +5,10 @@ RIGETTI_ARN = "arn:aws:braket:::device/qpu/rigetti/Aspen-11"
 IONQ_ARN = "arn:aws:braket:us-east-1::device/qpu/ionq/Aria-1"
 SIMULATOR_ARN = "arn:aws:braket:::device/quantum-simulator/amazon/sv1"
 OQC_ARN = "arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy"
```

Comment:
Should we remove `OQC_ARN` from these tests?

Reply:
I mean, `DWAVE_ARN` is still here. 🤷 Up to you, happy to remove it.